tests/e2e/common.exp

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/expect -f
+# Copyright 2017 Mirantis
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+proc connect_ssh {vmname} {
```

Comment:
Sorry, maybe I was a bit unclear concerning our expect e2e test approach... Basically expect only makes sense for interacting with the serial console, and in case of ssh plain ssh commands in e2e.sh are probably enough. Or am I missing some advantages of expect here?..